sweetalert2.d.ts

```diff
@@ -452,12 +467,33 @@ declare module 'sweetalert2' {
 
         /**
          * A custom CSS class for the modal.
+         * If a string (or an array of strings) value is provided, the classname will be applied to the popup.
```

Comment:
Sorry @limonte, actually the array of strings change was meant for the values of the keys inside the object. So sorry about asking you to change it back again.

Reply:
No problem! Reverted.